docs/example-scenario/ai/movie-recommendations.md

```diff
@@ -0,0 +1,131 @@
+
+* [Azure Cosmos DB][cosmosdb] enables globally distributed and multi-model database storage.
+
+* [Azure Container Instances][aci] is used to deploy the trained models to web or app services, optionally using [Azure Kubernetes Service][aks].
+
+### Alternatives
+
+[Azure Databricks][databricks] is a managed Spark cluster where model training and evaluating is performed. You can set up a managed Spark environment in minutes, and [autoscale][autoscale] up and down to help reduce the resources and costs associated with scaling clusters manually. Another resource-saving option is to configure inactive [clusters][clusters] to terminate automatically.
+
+## Considerations
+
+### Availability
+
+Machine-learning-built apps are split into two resource components: resources for training, and resources for serving. Resources required for training generally do not need  high availability, as live production requests do not directly hit these resources. Resources required for serving need to have high availability to serve customer requests.
+
+For training, the DSVM is available in [multiple regions][regions] around the globe and meets the [service level agreement][sla] (SLA) for virtual machines. For serving, Azure Kubernetes Service provides a [highly available][ha] infrastructure. Agent nodes also follow the [SLA][sla-aks] for virtual machines.
+
+### Scalability
+
+If you have a large data size, you can scale your DSVM to shorten training time. You can scale a VM up or down by changing the [VM size][vm-size]. Choose a memory size large enough to fit your dataset in-memory and a higher vCPU count in order to decrease the amount of time that training takes.
+
+### Security
+
+This scenario can use Azure Active Directory to authenticate users for [access to the DSVM][dsvm-id], which contains your code, models, and (in-memory) data. Data is stored in Azure Storage prior to being loaded on a DSVM, where it is automatically encrypted using [Storage Service Encryption][storage-security]. Permissions can be managed via Azure Active Directory authentication or role-based access control.
+
+## Deploy this scenario
+
+**Prerequisites**: You must have an existing Azure account. If you don't have an Azure subscription, create a [free account][free] before you begin.
+
+All the code for this scenario is available in the [Microsoft Recommenders repository][github].
+
+Follow these steps to run the [ALS quickstart notebook][notebook]:
+
+1. [Create a DSVM][dsvm-ubuntu] from the Azure portal.
+
+2. Clone the repo in the Notebooks folder:
+
+    ```shell
+    cd notebooks
+    git clone https://github.com/Microsoft/Recommenders
+    ```
+
+3. Install the conda dependencies following the steps described in the [SETUP.md][setup] file.
+
+4. In a browser, go to your jupyterlab VM and navigate to `notebooks/00_quick_start/als_pyspark_movielens.ipynb`.
+
+5. Execute the notebook.
+
+## Related resources

```

Comment:
Did we standardize on this heading for example scenarios? The usual docs convention is "Next steps"